fastlane/Fastfile

```diff
@@ -0,0 +1,30 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
+
+platform :ios do
+  desc "Description of what the lane does"
+  lane :custom_lane do
+    # add actions here: https://docs.fastlane.tools/actions
+  end
+end
+
+lane :unit_tests do
+  run_tests(build_for_testing: true,
+            scheme: "Basic-Car-Maintenance",
+            only_testing: "Basic-Car-Maintenance-Tests",
+            device: "iPhone 14")
```

Comment:
Can we change this to iPhone 15 Pro? is it available?

Reply:
Let me check but it should be possible

Reply:
I updated the device config to iPhone 15 Pro and there was no problem.